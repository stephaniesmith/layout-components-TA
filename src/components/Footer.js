import Component from './Component';
// import styles from './Footer.css';

export default class Footer extends Component {
  render() {
    return `
      <section>
        Footer!
      </section>`;
  }
}
