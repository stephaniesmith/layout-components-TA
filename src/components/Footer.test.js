import Footer from './Footer';

describe('Footer Component', () => {
  it('should be a footer', () => {
    const footer = new Footer();

    expect(footer.render()).toMatchSnapshot();
  });
});
