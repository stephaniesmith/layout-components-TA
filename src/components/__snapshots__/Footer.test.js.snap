// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should be a footer 1`] = `
"
      <section>
        Footer!
      </section>"
`;
